cmake/modules/Logging.cmake

```diff
@@ -17,6 +17,15 @@
 
 # This script configures the logging module and dependency on libbacktrace
 
+if(USE_CUSTOM_LOGGING)
+  # Set and propogate TVM_LOG_CUSTOMIZE flag is custom logging has been requested
+  target_compile_definitions(tvm_objs PUBLIC TVM_LOG_CUSTOMIZE=1)
```

Comment:
what about tvm_allvisible?

Reply:
I've added a change to propagate compile definitions from tvm to tvm_allvisible.

Reply:
🙏